### DISCUSSION TITLE
Release 3.3.0 / Spring Boot 3.4

### DISCUSSION THREAD

**Discussion by matthias-fritschi-bit** [awspring/spring-cloud-aws#1291] [General]
Hi

First of all, thanks for this awesome collection of starters!

Is there already a timeline for the 3.3.0 release? Or anything I could contribute to make it happen?

Thanks,
Matt

**Comment by maciejwalkowiak** [comment]
Hi! We're on it, it will happen this month.

**Reply by jesjos** [reply]
Is there an update on this? Is there something the community can do to help speed up the process?

**Comment by maciejwalkowiak** [comment]
@matthias-fritschi-bit @jesjos 3.3.0 has been released https://github.com/awspring/spring-cloud-aws/releases/tag/v3.3.0
